packages/cardpay-sdk/sdk/token-bridge-foreign-side.ts

```diff
@@ -140,27 +171,38 @@ export default class TokenBridgeForeignSide implements ITokenBridgeForeignSide {
     return estimatedGasInWei.divRound(rounder).mul(rounder);
   }
 
+  async claimBridgedTokens(txnHash: string): Promise<TransactionReceipt>;
   async claimBridgedTokens(
     messageId: string,
     encodedData: string,
     signatures: string[],
     txnOptions?: TransactionOptions,
     contractOptions?: ContractOptions
+  ): Promise<TransactionReceipt>;
+  async claimBridgedTokens(
+    messageIdOrTxnHash: string,
+    encodedData?: string,
+    signatures?: string[],
+    txnOptions?: TransactionOptions,
+    contractOptions?: ContractOptions
   ): Promise<TransactionReceipt> {
+    if (!encodedData) {
```

Comment:
this one is a little harder because a message ID looks exactly like a txnHash--they are both keccak256 hashes. so i don't think you can change this.